UnitTests/PaymentsCoreTests/NetworkingTests/RequestTests/AccessTokenRequestTests.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  APIRequestTests.swift
+//  APIRequestTests
+//
+//  Created by Jones, Jon on 9/20/21.
+//
+
+import Foundation
+@testable import PaymentsCore
+import XCTest
+
+class AccessTokenRequestTests: XCTestCase {
+    func testAccessTokenRequest_toURLRequest_vendsCorrectURLString() throws {
```

Comment:
I think we want `<testSUT_condition_result>` for naming. So for this something like:

`testToURLRequest_withValidURLComponents_vendsCorrectURLString`.